src/Components/Config/ConfigScreens/ServerScreen.js

```diff
@@ -68,6 +72,8 @@ class ServerScreen extends Component {
     //   !ganachePortStatus.pid[0].name.toLowerCase().includes('ganache') &&
     //   !ganachePortStatus.pid[0].name.toLowerCase().includes('electron')
 
+    const networkInterfaces = os.networkInterfaces()
```

Comment:
Yeah, this needs to live in the main process - interfaces should be passed in via IPC